### PR TITLE
UXA fixes for executor icon alignment

### PR DESCRIPTION
**Pull request by owillborn**
*Issue #:* /bcgov/entity#15576

*Description of changes:*
- Minor UXA tweaks to properly align new icons

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue [hunk 1]
```diff
@@ -628,11 +634,17 @@ export default defineComponent({
...
     margin-top: -3px !important;
     height: 22px !important;
     width: 22px !important;
   }
+  .business-executor-icon {
+    margin-top: -8px !important;
+    margin-left: -4px !important;
+    height: 29px !important;
+    width: 28px !important;
+  }
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue:647]: the svgs a little different in size? 
Kind of annoying i imagine 🙄

**Review by cameron-eyds — APPROVED** [review]
Looks good! 1 minor comment you can disregard haha

File: ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue (hunk 1 above)
Comment (owillborn) [reply ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue:647]: Very annoying

Comment (owillborn) [reply ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue:647]: ones square and the other isn't, thats why all the gross css is needed

Comment (cameron-eyds) [reply ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue:647]: Oof

**Review by dimak1 — APPROVED** [review]

**Review by doug-lovett — APPROVED** [review]

**Pull request merged**